android/sound.h

```diff
@@ -37,12 +37,13 @@ class CSound : public CSoundBase, public oboe::AudioStreamCallback
 {
     Q_OBJECT
 
-public:
+  public:
     static const uint8_t RING_FACTOR;
-    CSound ( void           (*fpNewProcessCallback) ( CVector<short>& psData, void* arg ),
+    CSound ( void ( *fpNewProcessCallback ) ( CVector<short>& psData,
```

Comment:
Well, the syntax appears to have beaten the formatter here!  The identifier, `fpNewProcessCallback`, should be aligned with `arg` and `strMIDISetup` below.  (This declarations `psData` and `arg` are, I think, redundant?)

Reply:
looks different now with `ColumnLimit: 120` - but not sure if better